lib/go-tc/deliveryservices.go

```diff
@@ -179,6 +188,11 @@ type DeliveryServiceNullableV30 struct {
 	ServiceCategory    *string `json:"serviceCategory" db:"service_category"`
 }
 
+type DeliveryServiceV31 struct {
```

Comment:
Do we need to signal if this, too, is `nullable` or not, like the nested struct?

Reply:
I believe the general rule is that new structs moving forward should be nullable, so it's not really necessary to call that out for new structs anymore.